nemo/collections/nlp/data/language_modeling/megatron/megatron_batch_samplers.py

```diff
@@ -120,78 +158,51 @@ def __iter__(self):
             yield batch[start_idx:end_idx]
 
 
+# NOTE (mkozuki): I haven't tested this enough.
+@experimental
```

Comment:
the decorator goes before `__init__`

Reply:
I think this is okay given the following two
- https://github.com/NVIDIA/NeMo/blob/ec99f425c5dd3458b3012f951677560ff9046d8d/nemo/utils/decorators/experimental.py#L21-L33
- https://github.com/NVIDIA/NeMo/blob/ec99f425c5dd3458b3012f951677560ff9046d8d/nemo/collections/cv/losses/nll_loss.py#L24-L25

Reply:
@titu1994 is it okay to decorate the class itself rather than the `__init__` ?